lib/src/common/variables.js

```diff
@@ -12,7 +12,10 @@ export const globalTokens = {
   hal_grey_s_40: "#666666",
   hal_black: "#000000",
   color_grey_800: "#4d4d4d",
+  color_grey_700: "#666666",
   color_grey_600: "#808080",
```

Comment:
`#666666` already exists in `hal_grey_s_40`.

Reply:
ok, i'll chage it. As we discuss, names are quite confusing